deltachat-ios/Controller/FilesViewController.swift

```diff
@@ -302,11 +301,18 @@ extension FilesViewController {
             objectsToShare = [fileURL]
         }
 
-        activityVC = UIActivityViewController(activityItems: objectsToShare, applicationActivities: nil)
+        let activityVC = UIActivityViewController(activityItems: objectsToShare, applicationActivities: nil)
         activityVC.excludedActivityTypes = [.copyToPasteboard]
         activityVC.popoverPresentationController?.sourceView = sourceView
         parentViewController.present(activityVC, animated: true, completion: nil)
     }
+
+    public static func share(url: String, parentViewController: UIViewController) {
+        if let url = URL(string: url) {
+            let activityVC = UIActivityViewController(activityItems: [url], applicationActivities: nil)
+            parentViewController.present(activityVC, animated: true, completion: nil)
+        }
+    }
```

Comment:
What's the connection between `FilesViewController` and sharing QR-code? So does this sharing-code live in `FilesViewController`, which seems to be responsible to show files?

Reply:
the only connection is that the other "share" function is there as well :)

keeping that together makes sense, but both could also go to Utils or so